portworx-k8-snapshots/index.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "Portworx Kubernetes Snapshots Practice Lab",
+  "description": "In this lab, we will get hands on with the creating volume snapshots and  restore operations in Kubernetes ",
```

Comment:
there is extra space in between and & restore `volume snapshots and  restore`

Reply:
fixed